packages/core/src/lib/keep_alive_manager.ts

```diff
@@ -89,7 +84,7 @@ export class KeepAliveManager {
     }
 
     if (this.relayKeepAliveTimers.has(peerId)) {
-      clearInterval(this.relayKeepAliveTimers.get(peerId));
+      this.relayKeepAliveTimers.get(peerId)?.map(clearInterval);
```

Comment:
This probably will introduce a but since code always does 
`this.relayKeepAliveTimers.set(peerId, interval);`